test_runner/src/main/kotlin/ftl/args/yml/errors/ErrorNodeResolver.kt

```diff
@@ -0,0 +1,23 @@
+package ftl.args.yml.errors
+
+import com.fasterxml.jackson.databind.JsonNode
+
+internal class ErrorNodeResolver {
```

Comment:
no state, no constructor args, no needs for class :).

Reply:
Thanks for this! I made changes :)